### PR TITLE
WIP test dummy 

### PR DESCRIPTION
**Pull request by PaweuB**
This is a test

### CODE REVIEW

**Comment by PaweuB** [comment]
test

**Comment by PaweuB** [comment]
closed

**Comment by PaweuB** [comment]
test

File: concent_api/core/message_handlers.py [hunk 1]
```diff
@@ -501,6 +501,7 @@ def handle_send_force_get_task_result(
 def handle_send_force_subtask_results(
     client_message: message.concents.ForceSubtaskResults
 ) -> Union[message.concents.ServiceRefused, message.concents.ForceSubtaskResultsRejected, HttpResponse]:
+    logger.info('>>>>>>>>> start handle_send_force_subtask_results')
```
Comment (Jakub89) [inline concent_api/core/message_handlers.py]: Typo

**Pull request closed without merging**